src/components/Navbar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';  
+
+export default function Navbar() {
+    return (
+        <nav className="topbar">
+            <div class="logoDiv">
```

Comment:
Make sure to use `className` as that's how we use it in React instead of `class`